lib/resources/datav2/rest_v2.js

```diff
@@ -129,10 +129,42 @@ async function getLatestQuote(symbol, config) {
   }
 }
 
+async function getSnapshot(symbol, config) {
+  const resp = await dataHttpRequest(
+    `/v2/stocks/${symbol}/snapshot`,
+    {},
+    config
+  );
+
+  return entityv2.AlpacaSnaphotV2(resp.data);
```

Comment:
Just a note to talk about error handling and timeouts

Reply:
ya it might be more user friendly catch the error and parse it for the user but I don't think the other endpoints do this and it should be consistent